data/swio/import-loss-shp.sh

```diff
@@ -0,0 +1,11 @@
+if [ $# -lt 1 ] 
+then
+	echo "Usage $0 <shapefile>"
+	exit 1
+fi
+
+filename=$1
+table_name=$(basename $filename .shp)
+
+echo "shp2pgsql -c -W UTF-8 -s 4326 $filename temp.$table_name  > $table_name.sql"
+shp2pgsql -c -W UTF-8 -s 4326 $filename temp.$table_name  > $table_name.sql
```

Comment:
The script will not work if the shp has spaces in the name.

Either we consider this ok or you need to put `$filename` inside double quotes